modules/update-cvo-conditions.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-update-service.adoc
+
+[id="update-cvo-conditions_{context}"]
+= Understanding CVO conditions
+
+The `ClusterVersion` object sets the conditions that describe the state of the cluster-version operator (CVO).
+
+Failing:: When `Failing` is set to True, the CVO fails to reconcile the cluster with the desired release image. In all cases, the impact on the cluster is that the dependent nodes in the manifest graph may not be reconciled. The graph may be flattened, in which case there are no dependent nodes.
++
+Most reconciliation errors result in `Failing=True`, even though `ClusterOperatorNotAvaliable` has special handling.
+
+NoDesiredImage:: The CVO has not been given a release image to reconcile. When NoDesiredImage is True, it is a CVO coding error. The `desiredUpdate` field returns you to the current CVO's release image.
+
+ClusterOperatorNotAvailable:: `ClusterOperatorNotAvailable` is set when the CVO fails to retrieve the ClusterOperator from the cluster or when the retrieved ClusterOperator does not satisfy the reconciliation conditions.
++
+Unlike most manifest-reconciliation failures, this error does not immediately result in `Failing=True`. Under some conditions during installs and updates, the CVO will treat this condition as a `Progressing=True` condition and give the operator up to forty minutes to level before reporting `Failing=True`.
+
+RetrievedUpdates:: When `RetrievedUpdates` is set to True, the CVO successfully retrieves updates. When the `RetrievedUpdates` is set to False, the cluster is not able to retrieve recommended updates. When a CVO is unable to retrieve recommended updates the `CannotRetrieveUpdates` alert displays the reason for not retrieving recommended updates.
```

Comment:
nit: "a CVO" -> "the CVO".  Each cluster only has one.

And [the `NoChannel` case][1] has the CVO not attempting to retrive recommended updates.  I'm not clear on whether that counts as "unable", but it will have `RetrievedUpdates=False` without `CannotRetrieveUpdates` firing.  Your current wording here doesn't point out that possibility.  We recently landed #22252 with some discussion of channel clearing for this purpose, although we didn't get down into the `RetrievedUpdates` level of detail there.

[1]: https://github.com/openshift/cluster-version-operator/blob/c07975c452303469c378bc5b0f70d8763eaca1f3/docs/user/status.md#retrievedupdates